pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -878,17 +716,15 @@ spec:
 
 	taskruns := []*v1beta1.TaskRun{
 		taskRunSuccess, taskRunWithSaSuccess,
-		taskRunSubstitution, taskRunInputOutput,
```

Comment:
why was taskRunSubstitution remvoed?

Reply:
I thought it mainly was related with the git resources, when deleting the resources these taskRuns just become meaningless?

Reply:
restored, thanks for the catch